src/arch/or1k.rs

```diff
@@ -137,7 +137,7 @@ pub unsafe fn swap(arg: usize, old_sp: &mut StackPointer, new_sp: &StackPointer,
         l.jr    r9
         l.nop
       "#
-      : : : "memory" : "volatile")
+      : : : : "volatile")
```

Comment:
I don't believe this is correct. This code does, in fact, access memory other than passed to it through input operands.